Formula/vowpal-wabbit.rb

```diff
@@ -12,23 +15,24 @@ class VowpalWabbit < Formula
     sha256 "23598d455b5a62bdf8f65df7b266b07be980738cfe69f664e5f1cb110ec72cf1" => :el_capitan
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  depends_on "cmake" => :build
   depends_on "boost"
 
   def install
     ENV.cxx11
-    ENV["AC_PATH"] = "#{HOMEBREW_PREFIX}/share"
-    system "./autogen.sh", "--prefix=#{prefix}",
-                           "--with-boost=#{Formula["boost"].opt_prefix}"
-    system "make"
-    system "make", "install"
+    # The project provides a Makefile, but it is a basic wrapper around cmake
+    # that does not accept *std_cmake_args.
+    # The following should be equivalent, while supporting Homebrew's standard args.
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
```

Comment:
I think we should add skip for examples/tests

Reply:
Could you clarify? I'm not seeing `examples/tests` in the build logs or the final installation. Is `-DBUILD_TESTS=OFF` what you're referring to? 

If so, I'll add that.

Reply:
Yep, that's the one. 👍

Reply:
Added. Thanks! Anything else?